content/md/first-steps/manage-data-quality.md

```diff
@@ -48,3 +45,36 @@ In the data quality panel of the product edit form, you can see which attribute
 
 ![Image data quality attribute groups reminder](../img/data-quality-attribute-groups-reminder.png)
 :::
+
+# Manage your dictionary
+Either because your product names are unique or due to industry acronyms that are not part of regular dictionaries, you can add words in your personal dictionary to fit your special needs and avoid miscalculation of your Quality Score.
+
+Several ways to add words to the `Dictionary`:
+- Through the "ignore" feature in the spell-check pop-in displayed when a possible typo error is detected
+- In the `Locales` settings screen
+
+![Image data quality spelling](../img/data-quality-spelling.png)
+:::
+
+:::info
+To save you time, in addition to the manual ways, Akeneo PIM searches all words used in your product data (in text, text area attributes and attribute and option labels. If a word that counts a minimum of 3 letters is found at least 10 times, then this word will be automatically added to your personal dictionary.)
+:::
+
+## Locales dictionary
+For users with rights to edit locales, it is possible to add, edit or delete words in the dictionary.
+Go to `Settings/Locales/Dictionary`
+
+![Image data quality dictionary](../img/data-quality-dictionary.png)
+:::
+
+Several ways to add words from the `Dictionary` screens
+- For several locales, by clicking on a locale name and using the mass action to add the same word or words for the selected locales
+- For a single  locale, by using the "add words" box
+
+![Image data quality add words to dictionary](../img/data-quality-add-words-dictionary.png)
+
+:::info
+To add several words at a time, separate them with comas "," or space.
+You can add up to 100 words at a time.
```

Comment:
This limit have been removed from the "TagInput" component in the DSM, so we can ignore this rule.

```suggestion
```